test/Constraints/rdar126583098.swift

```diff
@@ -0,0 +1,15 @@
+// RUN: %target-swift-emit-silgen %s -verify
```

Comment:
Since this is well-typed code, and for parameter packs its a good idea anyway to make sure it gets all the way through SILGen, you could just put it in test/SILGen/ even though the original bug report was a Sema issue.

Reply:
Sure, can move to SILGen 👍

Reply:
#79658